test-cloud/src/main/resources/templates-general.properties

```diff
@@ -4,5 +4,5 @@ kie.app.template.workbench=https://raw.githubusercontent.com/jboss-openshift/app
 kie.app.template.workbench-monitoring=https://raw.githubusercontent.com/jboss-openshift/application-templates/bpmsuite-wip/bpmsuite/bpmsuite70-businesscentral-monitoring.json
 kie.app.template.workbench-monitoring.smartrouter=https://raw.githubusercontent.com/jboss-openshift/application-templates/bpmsuite-wip/bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json
 kie.app.template.kie-server.database.external=https://raw.githubusercontent.com/jboss-openshift/application-templates/bpmsuite-wip/bpmsuite/bpmsuite70-executionserver-externaldb.json
-kie.app.template.smartrouter=https://raw.githubusercontent.com/jboss-openshift/application-templates/bpmsuite-wip/bpmsuite/bpmsuite70-smartrouter.json
+#kie.app.template.smartrouter=https://raw.githubusercontent.com/jboss-openshift/application-templates/bpmsuite-wip/bpmsuite/bpmsuite70-smartrouter.json
```

Comment:
(optional) Maybe comment why this is disabled and when can it be reenabled?

Reply:
Good point, comment added.